src/content/learn/rendering-lists.md

```diff
@@ -767,6 +773,9 @@ img { width: 100px; height: 100px; border-radius: 50%; }
 A very attentive reader might notice that with two `filter` calls, we check each person's profession twice. Checking a property is very fast, so in this example it's fine. If your logic was more expensive than that, you could replace the `filter` calls with a loop that manually constructs the arrays and checks each person once.
 
 In fact, if `people` never change, you could move this code out of your component. From React's perspective, all that matters is that you give it an array of JSX nodes in the end. It doesn't care how you produce that array:
```

Comment:
You forgot to remove the english version here. Can you remove lines from 767 - 769